srcs/server/server.cpp

```diff
@@ -19,9 +20,7 @@ namespace ft
 	{
 		while (1)
 		{
-			if (recieve_request_())
-			{
-			}
+			recieve_request_();	
```

Comment:
looks like we don't need this to be true/false because we just use try catch inside recieve_request_